fast64_internal/sm64/sm64_geolayout_classes.py

```diff
@@ -1,9 +1,52 @@
-from .sm64_geolayout_constants import *
-from .sm64_geolayout_utility import *
-from .sm64_constants import *
+import bpy
+from struct import pack
+from copy import copy
```

Comment:
this change results in an error on line 142.
copy.copy needs to be changed to copy